reviewers.js

```diff
@@ -45,6 +52,11 @@ function getReviewerByName(reviewerName, reviewers) {
  ****************************************************************/
 function getReviewerByReviewTitle(reviewTitle, reviewers) {
   //TODO: ADD YOUR CODE HERE
+  return reviewers.find((items) => {
+    items.some((value) => {
+      return value.books === reviewTitle;
```

Comment:
the condition for getting the reviewer by the title is incorrect, you should do this instead:
```
return reviewers.find((items) => {
  items.some((value) => {
     return value.title === reviewTitle;
   } 
}
```
Which is basically using the `reviewerHasReview` function, so you can use that instead:
```
  return reviewers.find((reviewer) => reviewerHasReview(reviewTitle, reviewer));
```